diff --git a/pyproject.toml b/pyproject.toml
index ee366e5..c11cfea 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -33,7 +33,7 @@ license = "MIT"
 
 [tool.poetry.dependencies]
 python = ">=3.8"
-prompt_toolkit = ">=2.0,<=3.0.36"  # once https://github.com/prompt-toolkit/python-prompt-toolkit/issues/1726 is fixed, this can be changed to ">=2.0,<4.0"
+prompt_toolkit = ">=2.0,<4.0"  # with versions >3.0.36 expect failures caused by https://github.com/prompt-toolkit/python-prompt-toolkit/issues/1726
 
 [tool.poetry.group.docs]
 optional = true
